sdk/monitor/tests.yml

```diff
@@ -0,0 +1,24 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      ServiceDirectory: monitor
+      TestTimeoutInMinutes: 300
+      BuildTargetingString: opentelemetry-exporter-azuremonitor
+      EnvVars:
+        AZURE_SUBSCRIPTION_ID: $(azure-subscription-id)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_TEST_RUN_LIVE: 'true'
+      Matrix:
+        Linux_Python35:
+          OSVmImage: 'ubuntu-18.04'
```

Comment:
Is there any reason for not running live test on default matrix?

Reply:
this package will not be python 2 compatible